schema/addresses/address.yaml

```diff
@@ -71,6 +71,14 @@ properties:     # JSON Schema: Top-level object properties.
         items: { "$ref": "#/$defs/propertyContainers/addressLevelContainer" }
         minItems: 1
         maxItems: 5
+      postal_city:
+        description: >-
+          In some countries or regions, a mailing address may need to specify a
```

Comment:
Should we mention anything about relation between `postcode` and `postal_city`? Probably safer to just ignore since it probably not true everywhere?

Reply:
What do you have in mind? At least in the US they are not necessarily related I don't believe

Reply:
Ye, this is OK as is, ignore me.